### PR TITLE
allow installation with symfony 6

### PR DESCRIPTION
**Pull request by dbu**
| Q             | A
| ------------- | ---
| Branch?       | "master"
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,39 +14,37 @@
...
-        "symfony/security-bundle": "^4.4 || ^5.0",
-        "symfony/phpunit-bridge": "^4.4.34 || ^5.0",
+        "ext-dom": "*",
+        "jackalope/jackalope-doctrine-dbal": "^1.3 || ^2.0",
         "mockery/mockery": "^1.4.1",
-        "symfony-cmf/routing-bundle": "^2.1.0",
-        "symfony-cmf/testing": "^4.0.0",
-        "doctrine/dbal": "^2.5",
+        "symfony-cmf/routing-bundle": "dev-sf7 as 3.1.0",
+        "symfony-cmf/testing": "dev-sf7 as 5.0.0",
```
Comment (dbu) [inline composer.json:26]: adjust once those are merged and tagged

**Comment by acrobat** [comment]
@dbu is there anything I can help with to move the PR forward?

**Comment by acrobat** [comment]
Ok, I will try to take a look at the failing tests!

**Comment by dbu** [comment]
after rebasing, the failure is gone :tada: 

did some more cleanups and released https://github.com/symfony-cmf/core-bundle/releases/tag/3.0.0

let me know if anything fails (or do fix PRs) but the CI is happy, so i tagged 3.0.0 already :crossed_fingers:

**Pull request merged**